### PR TITLE
Docs fix on object types

### PR DESCRIPTION
**Pull request by diksipav**

### CODE REVIEW

File: docs/datamodel/objects.rst [hunk 1]
```diff
@@ -11,8 +11,8 @@ analogous to SQL *tables* or ORM *models*, and consist of :ref:`properties
 Properties are used to attach primitive data to an object type. In versions
 prior to EdgeDB 3.0 they must be declared with the ``property`` keyword, but
 the keyword is not required in 3.0+ unless the property is a :ref:`computed
-property <ref_datamodel_computed>`. For the full documentation on properties,
-see :ref:`Properties <ref_datamodel_props>`.
+property <ref_datamodel_computed>` (from 4+ is not required for computeds neither).
+For the full documentation on properties, see :ref:`Properties <ref_datamodel_props>`.
```
Comment (scotttrinh) [inline docs/datamodel/objects.rst:15]: Actually, we should drop all of the language around the ``property`` keyword. We can show computed properties in the code blocks if we want to be thorough, but having all three historic styles being called out here is totally unnecessary.

**Review by scotttrinh — CHANGES_REQUESTED** [review]
Thanks for jumping in here! I think we should remove all of the language around the keywords for both properties and links.

File: docs/datamodel/objects.rst (hunk 1 above)
Comment (diksipav) [reply docs/datamodel/objects.rst:15]: Hm ok, I am fine with deleting these altogether. Will do it now

Comment (diksipav) [reply docs/datamodel/objects.rst:15]: I wasn't sure did u mean to write:
```
Properties are used to attach primitive data to an object type. In versions
prior to EdgeDB 3.0 they must be declared with the ``property`` keyword, but
the keyword is not required in 3.0+. For the full documentation on properties...
```

or to completely remove text about property:
```
Properties are used to attach primitive data to an object type. 
For the full documentation on properties...
```
I understood we want the 2nd.

**Review by scotttrinh — APPROVED** [review]
Yep, this is perfect!

**Pull request merged**